docs/en/ingest-management/faq.asciidoc

```diff
@@ -227,3 +226,27 @@ and port for your setup. If you run everything locally, the address is
 you can copy the {es} endpoint URL from the overview page of your deployment.
 If you're not running in {ecloud}, make sure the {kib} and {es} HTTPS ports
 are both accessible; by default these are `5601` and `9200` respectively.
+
+[discrete]
+[[how-do-i-reset-a-missing-dashboard-asset]]
+== If I delete an integration dashboard asset from {kib}, how do I get it back?
+
+To reinstall the assets for a specific integration, you can use the API.
+You'll need to know the package name and version you want to reinstall.
```

Comment:
```suggestion
In order to use the API, you'll need to know the package name and version
you want to reinstall.
```

Reply:
I think the introductory phrase is already implied, but feel free to commit if you think it makes things more clear.

Reply:
it only flows right to me, based on the addition of the other.  I like it. Will do, thanks.